src/npc.cpp

```diff
@@ -1712,7 +1712,15 @@ int npc::follow_distance() const
           g->m.has_flag( TFLAG_GOES_UP, g->u.pos() ) ) ) {
         return 1;
     }
-    // TODO: Allow player to set that
+    // Uses ally_rule follow_distance_4 to determine if should follow by 2 or 4 tiles
+    if( rules.has_flag(
```

Comment:
No, if the rule is follow_distance_4, you return a follow distance of 4.

I don't care what the variable name is, but the name has to match the use.